### DISCUSSION TITLE
Using an Existing VPC (CIDR block)

### DISCUSSION THREAD

**Discussion by dayekaye** [mozilla/hubs#3276] [Hubs Cloud Questions] [answered]
Hi, I am setting up a new Enterprise Hubs Cloud using the template. I am at the section where it is asking for a VPC. In the past, I've left that at its default state, but this time I want to use an existing VPC. If I enter my existing CIDR block in the Hub creation page line for VPC, will that cause a conflict with my existing VPC? or will it be able to add the 2 new subnets (from Hubs Cloud) without conflict?

**Comment by robinkwilson** [comment, marked as answer]
Is this a stack you already have running? So you'd attempt this with an update?

For the **Advanced** section of the template (editing the VPC), you should do this with a fresh new stack, _NOT_ with an update.

Follow instructions for a [backup and restore](https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html), so you can save the data + scenes + avatars you have on your existing stack while adding in your CIDR block for VPC.
